default_settings.json

```diff
@@ -0,0 +1,5254 @@
+{
+	// Controls whether the editor shows CodeLens.
+	"diffEditor.codeLens": false,
+
+	// When enabled, the diff editor ignores changes in leading or trailing whitespace.
+	"diffEditor.ignoreTrimWhitespace": true,
+
+	// Timeout in milliseconds after which diff computation is cancelled. Use 0 for no timeout.
+	"diffEditor.maxComputationTime": 5000,
+
+	// Maximum file size in MB for which to compute diffs. Use 0 for no limit.
+	"diffEditor.maxFileSize": 50,
+
+	// Controls whether the diff editor shows +/- indicators for added/removed changes.
+	"diffEditor.renderIndicators": true,
+
+	// Controls whether the diff editor shows the diff side by side or inline.
+	"diffEditor.renderSideBySide": true,
+
+	// 
+	//  - off: Lines will never wrap.
+	//  - on: Lines will wrap at the viewport width.
+	//  - inherit: Lines will wrap according to the `editor.wordWrap` setting.
+	"diffEditor.wordWrap": "inherit",
+
+	// Controls whether suggestions should be accepted on commit characters. For example, in JavaScript, the semi-colon (`;`) can be a commit character that accepts a suggestion and types that character.
+	"editor.acceptSuggestionOnCommitCharacter": true,
+
+	// Controls whether suggestions should be accepted on `Enter`, in addition to `Tab`. Helps to avoid ambiguity between inserting new lines or accepting suggestions.
+	//  - on
+	//  - smart: Only accept a suggestion with `Enter` when it makes a textual change.
+	//  - off
+	"editor.acceptSuggestionOnEnter": "on",
```

Comment:
I think this should be `false`, so that attempting to go to a new line doesn't accidentally accept suggestions. (I tried `smart` but couldn't tell what it was doing.)